tests/conftest.py

```diff
@@ -133,6 +133,31 @@ def fin():
 
     return pool
 
+
+@pytest.fixture(scope='session')
+def db_engine(request, loop):
+    import server
+
+    def opt(val):
+        return request.config.getoption(val)
```

Comment:
Will that function return `None`? If yes, then `password=pw or None` doesn't make sense

Reply:
I think it's there in case password is set to `''`